radsat-sk/framework/fileio/RFram.c

```diff
@@ -16,6 +16,13 @@
  * @brief Initialize the FRAM peripheral driver.
  */
 void framInit(void) {

```

Comment:
Your function is returning a void but in your code, it is returning an integer which is a compiler error. Maybe contact me if you are confused on the actual purpose of "void" in the function